pkg/rulesConfig/defaultRules/defaultRules.go

```diff
@@ -0,0 +1,58 @@
+package defaultRules
```

Comment:
I didn't understand why the package is named defaultRules?
what about `inCodeRules`?